cmd/rook/rgw.go

```diff
@@ -42,7 +43,8 @@ var (
 func init() {
 	rgwCmd.Flags().StringVar(&rgwName, "rgw-name", "", "name of the object store")
 	rgwCmd.Flags().StringVar(&rgwKeyring, "rgw-keyring", "", "the rgw keyring")
-	rgwCmd.Flags().StringVar(&rgwHost, "rgw-host", "", "dns host name")
+	rgwCmd.Flags().StringVar(&rgwHost, "rgw-host", "", "RGW hostname")
```

Comment:
Can you tell me what is the difference between the two config? rgw-host vs rgw-dns-name?. Do we still need the formal?

Reply:
The hostname is still set in the rgw properties, so both are still required. Thanks for the feedback!!